extensions/applicationinsights-clickanalytics-js/src/common/Utils.ts

```diff
@@ -0,0 +1,336 @@
+/**
+ * @copyright Microsoft 2020
+ * File containing utility functions.
+ */
+
+import { CoreUtils, _InternalMessageId, hasDocument } from "@microsoft/applicationinsights-core-js";
+import {
+    IClickAnalyticsConfiguration
+} from '../Interfaces/Datamodel';
+
+
+const Prototype = 'prototype';
+const DEFAULT_DONOT_TRACK_TAG = 'ai-dnt';
+const DEFAULT_AI_BLOB_ATTRIBUTE_TAG = 'ai-blob';
+const DEFAULT_DATA_PREFIX = 'data-';
+
+export const _ExtendedInternalMessageId = {
+    ..._InternalMessageId,
+    CannotParseAiBlobValue: 506,
```

Comment:
Can we just add these to the internal messages, otherwise we could end up duplicating these across different plugins (external and internal)

Reply:
Added these to AI Core internal messages. Will remove these here once we publish to npm